.travis.yml

```
@@ -1,8 +1,32 @@
+conditions: v1
 language: python
-sudo: required
-dist: xenial
-python: "3.7"
-cache: pip
+python:
+  - "3.6"
```

So just put 3.7 back here:
```suggestion
  - "3.6"
  - "3.7"
dist: xenial
```